client_test/types_test.go

```diff
@@ -0,0 +1,91 @@
+package client_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	"github.com/statechannels/go-nitro/client/engine/messageservice"
+	p2pms "github.com/statechannels/go-nitro/client/engine/messageservice/p2p-message-service"
+	"github.com/statechannels/go-nitro/internal/testactors"
+)
+
+const (
+	MAX_PARTICIPANTS       = 4
+	STORE_TEST_DATA_FOLDER = "../data/store_test"
+	ledgerChannelDeposit   = 5_000_000
+	defaultTimeout         = 10 * time.Second
+	virtualChannelDeposit  = 5000
+	DURABLE_STORE_FOLDER   = "../data/client_test"
+)
+
+type StoreType string
+
+const (
+	MemStore     StoreType = "MemStore"
+	DurableStore StoreType = "DurableStore"
+)
+
+type ChainType string
+
+const (
+	MockChain      ChainType = "MockChain"
+	SimulatedChain ChainType = "SimulatedChain"
+)
+
+type TestParticipant struct {
+	StoreType StoreType
+	Name      testactors.ActorName
```

Comment:
An alternative is for `TestParticipant` to contain the `testactors.Actor` (instead of just the name). That way we don't have "a foreign key" relationship from a TestParticipant to an actor.

Reply:
Good idea 👍

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/1193/commits/30bf516375af1918c2a983fc2b67d726df9ac6b9